src/Kaleidoscope.cpp

```diff
@@ -9,6 +11,9 @@ Kaleidoscope_::Kaleidoscope_(void) {
 
 void
 Kaleidoscope_::setup(void) {
+
+  kaleidoscope::Hooks::init();
```

Comment:
Hooks are now access via static functions of class `Hook`. The call to `init()` initializes all plugins. This is what formerly has been done through `Kaleidoscope.run()`.